app/controlplane/internal/biz/workflow.go

```diff
@@ -122,8 +122,13 @@ func (uc *WorkflowUseCase) findOrCreateContract(ctx context.Context, orgID, cont
 	}
 
 	// No contractID has been provided so we create a new one
-	contractName := fmt.Sprintf("%s/%s", project, name)
-	return uc.contractUC.Create(ctx, orgID, contractName, nil)
+	contractName := name
+	// Project might be empty
+	if project != "" {
+		contractName = fmt.Sprintf("%s-%s", project, name)
```

Comment:
It will append a suffix, note the `AddUniquePrefix: true` property below.

That enables a logic to retry the creation of a name with a 4 digits suffix, similar to what we do when we generate a workflow